common/utils/src/main/resources/error/error-classes.json

```
@@ -97,6 +97,13 @@
       "The method <methodName> can not be called on streaming Dataset/DataFrame."
     ]
   },
+  "CANNOT_ALTER_PARTITION_COLUMN" : {
+    "message" : [
+      "ALTER TABLE (ALTER|CHANGE) COLUMN is not supported for partition columns.",
+      "<columnName> is a partition column in table <tableName>."
```

Let's follow other message in `error-classes.json`:
```suggestion
      "ALTER TABLE (ALTER|CHANGE) COLUMN is not supported for partition columns, but found the partition column <columnName> in the table <tableName>."
```